InscryptionAPI/AscensionScreens/AscensionRunSetupScreenBase.cs

```diff
@@ -0,0 +1,433 @@
+using DiskCardGame;
+using HarmonyLib;
+using UnityEngine;
+using GBC;
+
+namespace InscryptionAPI.AscensionScreens
+{
+    public abstract class AscensionRunSetupScreenBase : ManagedBehaviour
+    {
+        // This will display near the top of the screen.
+        // The very top will be the challenge level text; below that will be this
+        public abstract string headerText { get; }
+
+        // If this is true, it will show the card displayer at the footer below the challenge text
+        // Seting this to true will also enable the DisplayCardInfo and ClearCardInfo methods
+        // If this is not true, those methods will do nothing
+        public abstract bool showCardDisplayer { get; }
+
+        // If this is true, it will put a card panel in the middle of the screen
+        // The card panel will be able to display up to 7 cards
+        // and will have a left/right button
+        public abstract bool showCardPanel { get; }
+
+        // This is the hook for the implementer to build their own UI elements.
+        public virtual void InitializeScreen(GameObject partialScreen)
+        {
+
+        }
+
+        public SequentialPixelTextLines cardInfoLines;
+
+        public SequentialPixelTextLines challengeFooterLines;
+
+        public ChallengeLevelText challengeHeaderDisplay;
+
+        public List<PixelSelectableCard> cards;
+
+        public GameObject cardPanel;
+
+        public MainInputInteractable leftButton;
+
+        public MainInputInteractable rightButton;
+
+        public AscensionMenuInteractable continueButton;
+
+        public AscensionMenuBackButton backButton;
+
+        public PixelText screenTitle;
+
+        public PixelText secondaryInfoDisplayer;
+
+        public static AscensionRunSetupScreenBase BuildScreen(Type screenType, AscensionMenuScreens.Screen previousScreen, AscensionMenuScreens.Screen nextScreen)
+        {
+            // Create the new screen
+            InscryptionAPIPlugin.Log.LogInfo($"Creating screen for {screenType.Name}");
```

Comment:
Maybe switch these to debug instead of Info? this seems like a lot of random info to give to every user, or maybe just get rid of them altogether

Reply:
I switched it from loginfo to logdebug